src/components/sections/communities/_partials/Header.tsx

```diff
@@ -19,22 +19,20 @@ export default function Header({
   isTeamChallenge = false,
   isHackathon = false,
 }: HeaderProps): ReactElement {
-  const headerClassName = classNames("text-4xl md:text-5xl leading-none", {
+  const headerClassName = classNames("text-4xl md:text-5xl leading-none text-gray-900", {
     "hidden md:flex": hideTitleOnMobile,
-    "text-gray-400": subtitle,
-    "text-gray-900": !subtitle,
   });
 
   return (
     <div>
       <h1 className={headerClassName}>{title}</h1>
       <div className="flex items-start gap-1">
         {subtitle && (
-          <h2 className="text-4xl flex items-center font-normal leading-none md:text-5xl text-default">
+          <h2 className={`text-4xl flex items-center font-normal leading-none md:text-5xl ${title? "text-gray-400": "text-gray-900"}`}>
```

Comment:
think it will be more readable to use `classNames` here

Reply:
classNames comes in handy when you have multiple condition classes , in this case where we only have a single condition, using a ternary operator directly in the className string is often simpler and more straightforward which is readable as well.